CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+# Contributing
+
+Please follow these steps:
+
+1. Fork this repository on GitHub
+1. Clone your fork
+1. Ensure all tests pass before making changes (`mvn test`)
+1. Make your changes on a topic branch off of master
+    - Write tests for your changes, and make sure *all* tests pass
+    - Commits should be atomic
+    - Commit messages should be in the following format:
+    
+            Short summary of the change
+        
+            Longer, more detailed message if needed. This may span multiple
+            paragraphs but each line should be no longer than 80 characters. 
```

Comment:
I generally try to keep things within that width, in case I'm running `git log` in a console. But consoles are usually wider than 80 chars these days and I can't remember the last time this was an issue, so I'll remove this.